Makefile

```diff
@@ -51,6 +51,7 @@ NVIDIA ?=0
 GPU ?= 0
 BUILD_NGINX ?= 0
 MEDIAPIPE_DISABLE ?= 0
+PYTHON_DISABLE ?= 1
```

Comment:
Should we check for incorrect setting such as:
`PYTHON_DISABLE=0 MEDIAPIPE_DISABLE=1`
?

Reply:
I think we could skip such check for now. I'd like to discuss if we even want to have such flag. Perhaps we could always build with Python enabled.